specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json

```diff
@@ -193,7 +193,51 @@
                         }
                     }
                 }
-            }
+            },
+            "patch": {
+              "tags": [
+                  "HanaOnAzure"
+              ],
+              "operationId": "HanaInstances_UpdateTags",
+              "summary": "Patches the Tags field of a SAP HANA instance.",
+              "description": "Patches the Tags field of a SAP HANA instance for the specified subscription, resource group, and instance name.",
+              "x-ms-examples": {
+                  "Get properties of a HANA instance": {
+                      "$ref": "./examples/HanaInstances_PatchTags.json"
+                  }
+              },
+              "parameters": [
+                  {
+                      "$ref": "#/parameters/ApiVersionParameter"
+                  },
+                  {
+                      "$ref": "#/parameters/SubscriptionIdParameter"
+                  },
+                  {
+                      "$ref": "#/parameters/ResourceGroupNameParameter"
+                  },
+                  {
+                      "$ref": "#/parameters/HanaInstanceNameParameter"
+                  },
+                  {
+                      "$ref": "#/parameters/HanaInstancePatchTagsParameter"
```

Comment:
How do other PATCH APIs do it? [Tags is already declared as a simple type](https://github.com/Azure/azure-rest-api-specs/blob/5ba20f526d6965bc6f9284175a740bfd6b9c43df/specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json#L264) in parts of our API definitions - can we keep it simple here?

Reply:
Fixed it

Reply:
I looked into other Patch Tags APIs, they all made a separate definition for tags. They just didn't name it "Tags". For example, in timeseriesinsights.json:
`"EventSourceUpdateParameters": {
      "type": "object",
      "properties": {
        "tags": {
          "type": "object",
          "additionalProperties": {
            "type": "string"
          },
          "description": "Key-value pairs of additional properties for the event source."
        }
      },
      "description": "Parameters supplied to the Update Event Source operation."
    },`

It also have Tags as part of a definition too. Which is similar to our `Resource` definition.
`"TrackedResource": {
      "properties": {
        "location": {
          "type": "string",
          "description": "Resource location",
          "x-ms-mutability":["read", "create"]
        },
        "tags": {
          "type": "object",
          "additionalProperties": {
            "type": "string"
          },
          "description": "Resource tags"
        }
      },`

I also tried to use the one defined under Resource, it's failing the checks with error: `Reference path 'Resource/properties/tags' does not exist in the definition section of the Swagger document.`